kernel/kernel-api/src/main/java/io/delta/kernel/internal/snapshot/SnapshotManager.java

```diff
@@ -580,4 +576,11 @@ private Optional<LogSegment> getLogSegmentWithMaxExclusiveCheckpointVersion(
         // TODO
         return Optional.empty();
     }
+
+    // TODO logger interface to support this across kernel-api module
+    private void logDebug(Supplier<String> message) {
```

Comment:
nice!

Reply:
+1 lets make this interface. its going to be internal interface right? so its okay if this is dont after 3.1.